src/Entities/ArrowEntity.cpp

```diff
@@ -125,7 +125,7 @@ void cArrowEntity::OnHitEntity(cEntity & a_EntityHit, Vector3d a_HitPos)
 	}
 
 	// Broadcast successful hit sound
-	GetWorld()->BroadcastSoundEffect("entity.arrow.hit_player", GetPosition(), 0.5, static_cast<float>(0.75 + (static_cast<float>((GetUniqueID() * 23) % 32)) / 64));
+	GetWorld()->BroadcastSoundEffect("entity.arrow.hit", GetPosition(), 0.5, static_cast<float>(0.75 + (static_cast<float>((GetUniqueID() * 23) % 32)) / 64));
```

Comment:
Maybe it should be `successful_hit`

Ref: http://www.theredengineer.com/1.9-playsound-list.html

Reply:
successful_hit is the previous name for hit_player

Reply:
Oh, I misunderstood. This sound is played at the arrow's position, not to the player who shot the bow.